src/components/Home/Home.js

```diff
@@ -15,18 +15,24 @@ class Home extends Component {
   }
 
   render(){
-    console.log('[Home Component][render]');
-    const groups = this.props.groupStatus && this.props.groupStatus.groups  ? this.props.groupStatus.groups.map(item => (
-      <Group 
-        key={item.nameUS}
-        title={item.nameUS}
-      />
-    )
-  ) : [];
+    console.log(this.props.groupStatus);
+    var groups = [];
+    if(this.props.groupStatus && this.props.groupStatus.groups) {   
+      _.mapValues(this.props.groupStatus.groups, (item) => {

```

Comment:
why you remove the group map? 
why you use _.mapValues? - why simple js map not working here?
why you create array with components?

Reply:
why lodash?

Reply:
Because now it's object, not array. It's need for certain reasons(create readable routes and easely finding group by group name), and lodash don't create new array like native js map method, so I create new array manually.

Reply:
And lodash, because lodash cool =)